### ISSUE TITLE
"LGBT (excluding pornography)"

### ISSUE THREAD

**Issue by ecawthon** [citizenlab/test-lists#314]
There are several different category codes (other than PORN) that could arguably include pornography, notably

> Provocative Attire,PROV,PROV,"Websites which show provocative attire and portray women in a sexual manner, wearing minimal clothing. "
> Sex Education,XED,XED,"Includes contraception, abstinence, STDs, healthy sexuality, teen pregnancy, rape prevention, abortion, sexual rights, and sexual health services."
> Online Dating,DATE,DATE,"Online dating services which can be used to meet people, post profiles, chat, etc"
> Media sharing,MMED,MMED,"Video, audio or photo sharing platforms."

The LGBT category, however, is alone in having a description that specifies that it excludes pornography:
> LGBT,LGBT,GAYL,A range of gay-lesbian-bisexual-transgender queer issues. (Excluding pornography)

This seems inconsistent. I propose that either "Excluding pornography" be added to the descriptions of PROV, XED, DATE, and MMED, or that it be removed from the description of LGBT.

**Comment by hellais** [comment]
Yes this is a very good point, thanks for suggesting it and raising it.

In order for this to be done, though we need to reprocess all the test list updating the description in each file to include this updated description.

I am thinking that perhaps this is a good opportunity to also take care of the issue of https://github.com/citizenlab/test-lists/issues/582.

**Comment by mhombach** [comment]
@hellais  Since 1.5 years have passed, what is the status of this issue? :)
